--- aap-1.090/main.aap	2009-01-18 23:53:47.712943686 +0200
+++ aap-1.091/main.aap	2010-01-05 07:58:12.308951140 +0200
@@ -47,13 +47,13 @@
 				    stdin).group(1) | :assign Version
 
 	# Directories to be used.
-	dir = $PREFIX/lib/aap/Exec-$Version
+	dir = $PREFIX/share/aap
 	toolsdir = $dir/tools
 	modulesdir = $dir/modules
 	docdir = $dir/doc
 	htmldocdir = $docdir$?HTMLDIR
 	bindir = $PREFIX/bin
-	mandir = $PREFIX/man/man1
+	mandir = $PREFIX/share/man/man1
 	sharedocdir = $PREFIX/share/doc
 	sharedocdiraap = $sharedocdir/aap
 	sharehtmldocdir = $sharedocdiraap$?HTMLDIR
